assets/js/admin/wpjm-modal.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
It was also extracted, just in order to make it a module instead of a global function.